model_security_adjust_cft/__openerp__.py

```diff
@@ -4,18 +4,22 @@
 
 {
     "name": "Model Security Adjust CFT",
-    "version": "8.0.1.0.0",
+    "version": "8.0.1.1.0",
     "author": "Ecosoft Co. Ltd.",
     "license": "AGPL-3",
     "description": """
     """,
     "category": "Uncategorized",
     "depends": [
-        'crm',
+        'crm','purchase','product_price_visible',
     ],
     "data": [
         'security/module_data.xml',
-        'data/ir.model.access.csv'
+        'views/partner_view.xml',
+        'security/product_security.xml',
+        'security/base_security.xml',
```

Comment:
Reorder so that security files go first.  Ref: https://www.odoo.com/forum/help-1/question/security-access-rights-error-for-a-custom-module-72983#answer_73052